parm/config/gfs/config.ocnanal

```diff
@@ -6,21 +6,15 @@
 echo "BEGIN: config.ocnanal"
 
 export OBS_YAML_DIR="${HOMEgfs}/sorc/gdas.cd/parm/soca/obs/config"
-export OBS_LIST=@SOCA_OBS_LIST@
-export OBS_YAML="${OBS_LIST}"
-export FV3JEDI_STAGE_YAML="${HOMEgfs}/sorc/gdas.cd/test/soca/testinput/dumy.yaml"
+export OBS_LIST=@SOCA_OBS_LIST@  # TODO(GA): doesn't look necessary as is to have
```

Comment:
Based on the comments able, can this become:
```suggestion
export OBS_LIST="${PARMgfs}/gdas/soca/obs/obs_list.yaml"
```
Same change is likely needed in `config.prepoceanobs`

Reply:
same comment as above, I want to have the option to point to a different obs list in that yaml.